### DISCUSSION TITLE
7.0.0

### DISCUSSION THREAD

**Discussion by hluk** [hluk/CopyQ#2310] [General]
## Added

- Windows installer has an option to install for current user or all users
  (#1912).

## Changed

- The preferred format to edit is now "text/plain;charset=utf-8" with
  "text/plain" as fallback. Additionally, if no such format is available,
  "text/uri-list" is used.

- Toggle Clipboard Storing menu item uses static text and icon instead of
  changing these dynamically after each use (#2255).

- Settings integrity is now handled solely by Qt. Previously, additional
  `*.bak` files where created for configuration files.

- Commands are no longer migrated to the new format on start. The old command
  configuration file has been last used in version 3.9.0 (released on
  2019-06-27).

- Native notification text length is limited now to avoid slow downs when
  showing notifications in some desktop environments. The limit is about
  100,000 characters and 100 lines.

## Fixed

- Fixes Sort/Reverse Selected Items menu actions (#2267).

- Fixes moving items to a tab in tab bar using drag'n'drop (#1246).

- Fixes possibly buggy window manager frame geometry (#2247).

<hr /><em>This discussion was created from the release <a href='https://github.com/hluk/CopyQ/releases/tag/v7.0.0'>7.0.0</a>.</em>

**Comment by boyanmolina** [comment]
Hi, Win 10 again gives "terrible" warnings when running the installation... Just so you know. Cheers and thanks for the awesome app.

**Reply by theAkito** [reply]
That is "normal".

**Reply by akoppa** [reply]
I updated a couple of minutes ago and got no warnings.

**Comment by nrbnlulu** [comment]
.deb is still version 6.0.1

**Comment by hongyumax** [comment]
Lack of font size adjustment
